ksql-functional-tests/src/test/resources/query-validation-tests/session-windows.json

```diff
@@ -91,10 +91,10 @@
       ],
       "outputs": [
         {"topic": "S2", "key": 0, "value": "0,0", "timestamp": 0, "window": {"start": 0, "end": 30000, "type": "time"}},
-        {"topic": "S2", "key": 0, "value": null, "timestamp": 10000, "window": {"start": 0, "end": 30000, "type": "time"}},
+        {"topic": "S2", "key": 0, "value": null, "timestamp": 0, "window": {"start": 0, "end": 30000, "type": "time"}},
         {"topic": "S2", "key": 0, "value": "0,5", "timestamp": 10000, "window": {"start": 0, "end": 40000, "type": "time"}},
         {"topic": "S2", "key": 1, "value": "1,100", "timestamp": 10000, "window": {"start": 10000, "end": 40000, "type": "time"}},
-        {"topic": "S2", "key": 1, "value": null, "timestamp": 40000, "window": {"start": 10000, "end": 40000, "type": "time"}},
+        {"topic": "S2", "key": 1, "value": null, "timestamp": 10000, "window": {"start": 10000, "end": 40000, "type": "time"}},
```

Comment:
This seems suspicious, too. `40000` seems to be correct

Reply:
as above.